### ISSUE TITLE
setupCMD ignored

### ISSUE THREAD

**Issue by cwchristerw** [Sam-DevZ/Discord-RoleReact#32]
**Describe the bug**
No matter, what you type to setupCMD bot will always answer when yourID is correct.

**To Reproduce**
1. Set setupCMD: "rr!test",
2. Start bot
3. Send message "rr!qwerty" or "The quick brown fox jumps over the lazy dog" with your account that has yourID
4. Bot will answer to your message even when you type wrong setupCMD

**Expected behavior**
It's expected to not answer when wrong setupCMD or any normal message is posted by yourID user :smile:

**Additional context**
config.js
```
module.exports = {

	/**
	 * Instructions on how to get this: https://redd.it/40zgse
	 */
	yourID: "[DEDUCTED]",

	setupCMD: "rr!test",

	/**
	 * Delete the 'setupCMD' command after it is ran. Set to 'true' for the command message to be deleted
	 */
	deleteSetupCMD: true,

	initialMessage: `**React to the messages below to receive the associated role. If you would like to remove the role, simply remove your reaction!**`,
	
	embedMessage: `
	React to the emoji that matches the role you wish to receive.
	
	If you would like to remove the role, simply remove your reaction!
	`,
	
	/**
	 * Must set this if "embed" is set to true
	 */
	embedFooter: "Role Group: Doer",
	
	roles: ["Gamer", "Coder"],

	/**
	 * For custom emojis, provide the name of the emoji
	 */
	reactions: ["👾", "💻"],

	/**
	 * Set to "true" if you want all roles to be in a single embed
	 */
	embed: true,

	/**
	 * Set the embed color if the "embed" variable is et to "true"
	 * Format:
	 * 
	 * #dd9323
	 */
	embedColor: "#dd9323",

	/**
	 * Set to "true" if you want to set a thumbnail in the embed
	 */
	embedThumbnail: false,

	/**
	 * The link for the embed thumbnail if "embedThumbnail" is set to true
	 */
	embedThumbnailLink: "",

	/**
	 * You"ll have to set this up yourself! Read more below:
	 * Please do not commit this token to the public if you contributed to this repository
	 * or host your code anywhere online. Giving someone your bot's token is the equivalent
	 * to giving someone the keys to your house and walking away!
	 * 
	 * https://github.com/reactiflux/discord-irc/wiki/Creating-a-discord-bot-&-getting-a-token
	 */
	botToken: "[DEDUCTED]"

};
```

---

If you want to implement a fix for this bug, please open a draft pull request which references this issue.

**Comment by cwchristerw** [comment]
This problem appeared after today I started to use newest version.

**Comment by cwchristerw** [comment]
This maybe caused by this line     if ((message.author.id !== CONFIG.yourID) && (message.content.toLowerCase() !== CONFIG.setupCMD)) return;

**Comment by cwchristerw** [comment]
I will fix this

**Comment by cwchristerw** [comment]
This will be fixed by #33

**Comment by acollierr17** [comment]
Hey, thanks for submitting this issue. 

Does this issue only happen with multiple instances of the bot running or only with one instance of the bot?

**Comment by cwchristerw** [comment]
This will happen on both scenarios you meantioned.

`if ((message.author.id !== CONFIG.yourID) && (message.content.toLowerCase() !== CONFIG.setupCMD)) return;` this will do only return when both are true and it means author.id and setupcmd has to be wrong.


`if ((message.author.id !== CONFIG.yourID) || (message.content.toLowerCase() !== CONFIG.setupCMD)) return;` My change will make sure that if eather of these is wrong it will return and dont do anything.

**Comment by acollierr17** [comment]
Thanks for the clarification. That does make sense. As you can see, I meant to check if both conditions returned true or not. But it would be better to return if only one of the conditions are true.

I will sort #33 out when I'm home.

**Comment by cwchristerw** [comment]
Closing this issue because it's fixed by #33
